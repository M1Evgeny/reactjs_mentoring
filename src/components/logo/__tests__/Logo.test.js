import React from "react";
import { render } from "@testing-library/react";
import { Logo } from "..";

describe("Logo test", () => {
  test("logo snapshot", () => {
    const { asFragment } = render(<Logo />);
    expect(asFragment(<Logo />)).toMatchSnapshot();
  });
});
