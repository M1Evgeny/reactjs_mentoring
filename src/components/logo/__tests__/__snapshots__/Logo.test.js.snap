// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo test logo snapshot 1`] = `
<DocumentFragment>
  <div
    class="undefined"
  >
    <b>
      netflix
    </b>
    roulette
  </div>
</DocumentFragment>
`;
